emData/download.sh

```diff
@@ -13,6 +13,7 @@ luts_url="https://cernbox.cern.ch/index.php/s/DuhCjcykSHZLRhM/download"
 declare -a processing_modules=(
   # VMRouter
   "VMR_L1PHIE"
+  "VMR_L2PHIB"
```

Comment:
Is this change necessary for the InputRouter?

Reply:
No - removing.